css/layout-css/news-feed-style.upload.css

```diff
@@ -1026,7 +1038,7 @@
   overflow-x: hidden;
   padding: 0 15px;
   margin: 54px 0 109px;
-  margin-bottom: calc(109px + env(safe-area-inset-bottom));
+  padding-bottom: env(safe-area-inset-bottom);
```

Comment:
Is this to fix the issue we saw earlier on?

Reply:
@tonytlwu Yes!